generators/server/templates/pom.xml.ejs

```diff
@@ -158,14 +159,23 @@
         <!-- Rule https://sonarcloud.io/coding_rules?open=squid%3AS4684&rule_key=squid%3AS4684 -->
         <sonar.issue.ignore.multicriteria.S4684.resourceKey><%= MAIN_DIR %>java/**/*</sonar.issue.ignore.multicriteria.S4684.resourceKey>
         <sonar.issue.ignore.multicriteria.S4684.ruleKey>squid:S4684</sonar.issue.ignore.multicriteria.S4684.ruleKey>
-        <sonar.jacoco.reportPaths>${project.testresult.directory}/coverage/jacoco/jacoco.exec</sonar.jacoco.reportPaths>
+        <sonar.coverageReportPaths>${project.testresult.directory}/TESTS-results-jest.xml</sonar.coverageReportPaths>
```

Comment:
Are you sure that works? I get the following with gradle `Error during parsing of the generic coverage report '/home/fred/git/github/jhipster/gradle-integration-test/build/test-results/TESTS-results-jest.xml'. Look at SonarQube documentation to know the expected XML format.`

Reply:
Actually no, I was not able to found any working example.
Will investigarem more in few hours...

Reply:
Looking at this file and the file in the `sonar` subfolder the one the sonar folder has the correct format: https://docs.sonarqube.org/latest/analysis/generic-test/

For gradle I use now `property "sonar.testExecutionReportPaths", "${project.buildDir}/test-results/jest/TESTS-results-sonar.xml"` and it works with latest sonar 7.6